runtime/tests/tests.rs

```diff
@@ -687,7 +696,7 @@ fn contract_call_fails_when_exchange_liquidity_is_low() {
 			});
 			initialize_block();
 			let r = Executive::apply_extrinsic(xt);
-			assert_eq!(r, Err(InvalidTransaction::Payment.into()));
+			assert_eq!(r, Err(InvalidTransaction::Custom(UNKNOWN_BUY_FEE_ASSET).into()));
```

Comment:
The purpose of this PR is to make errors known so I didn't expect this 'unknown' kind to actually be used 🤔 
Should be an 'insufficient exchange liquidity for fee payment'

Reply:
Yep I'm working on this right now lol thanks for the comment